src/js/view/view.js

```diff
@@ -186,6 +186,12 @@ function View(_api, _model) {
             _currentlyFloating = floating;
             _this.trigger(FLOAT, { floating });
             updateVisibility();
+
+            // On mobile Chrome an ad may receive the original resize event before wrapper is revealed.
+            // Listen for wrapper to be revealed
+            if (!containerHeight && !containerWidth) {
+                _this.responsiveListener();
+            }
```

Comment:
Why don't we avoid setting `containerWidth` and `containerHeight` to`0` while transitioning? Then when we arive at the end of the transition and are displaying again, this code can run without changes.

Reply:
https://github.com/jwplayer/jwplayer/pull/3837/files#r610730964